src/app/components/main-page/main-page.component.ts

```diff
@@ -3,7 +3,7 @@ import { Router, RouterEvent, NavigationEnd } from '@angular/router';
 import { AngularFireAuth } from '@angular/fire/auth';
 import { AngularFireStorage } from '@angular/fire/storage';
 
-import { faPlusSquare, faUser, faHome, faSearch, faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+import { faPlusSquare, faUser, faHome, faSearch, faEllipsisV, faDownload, faUpload } from '@fortawesome/free-solid-svg-icons';
```

Comment:
we can remove the icon here too if were not using the faUpload

Reply:
Yes

Reply:
fixed